### PR TITLE
include_role for infra roles

### PR DESCRIPTION
**Pull request by agonzalezrh**

### CODE REVIEW

File: ansible/configs/osp-migration/infra.yml [hunk 1]
```diff
@@ -185,13 +185,13 @@
     # Use  server.metadata.hostname  instead
     - name: Run infra-osp-create-inventory Role
       include_role:
-        name: infra-osp-create-inventory
+        name: "{{ANSIBLE_REPO_PATH}}/roles-infra/infra-osp-create-inventory"
```
Comment (wkulhanek) [inline ansible/configs/osp-migration/infra.yml]: Do we actually need the ANSIBLE_REPO_PATH ? Isn't that in the search path anyway?

Comment (agonzalezrh) [reply ansible/configs/osp-migration/infra.yml]: I copied from here:

./configs/linklight-foundations/post_infra.yml:      name: "{{ANSIBLE_REPO_PATH}}/roles-infra/infra-common-ssh-config-generate"

Maybe you are right and we don't needed it

Comment (wkulhanek) [reply ansible/configs/osp-migration/infra.yml]: @fridim  ?

Comment (agonzalezrh) [reply ansible/configs/osp-migration/infra.yml]: I modified it now, we can try without the path

**Comment by ezr-ondrej** [comment]
Hi, just wondering, wouldn't this be enough: #1299

**Comment by wkulhanek** [comment]
Right. That was my point. Should be in the "search path"

**Pull request closed without merging**